.github/workflows/build-deploy-pudl.yml

```diff
@@ -3,7 +3,7 @@ on:
   workflow_dispatch:
   push:
     tags:
-      - "v**"
+      - "v20*"
```

Comment:
Sadly it seems like we can't do a regex like `v\d{4}.\d{2}.\d{2`, just glob matching.

Reply:
Yeah, I looked around a bit too.